modules/programs/fish.nix

```
@@ -14,7 +14,7 @@ let
           Path to the plugin folder.
           </para><para>
           Relevant pieces will be added to the fish function path and
-          the completion path. The <filename>init.fish</filename> and
+          the completion path. The <filename>$initFile</filename> and
           <filename>key_binding.fish</filename> files are sourced if
```

```suggestion
          <filename>$keyBindingFile</filename> files are sourced if
```

Since we're already here, might as well make this configurable as well (there's at least one plugin I remember seeing that used `key-binding.fish` instead).

Also, I think the tags should be changed from `filename` to something else (maybe rycee knows what it should be; I'm not familiar with docbook) because `$initFile` is no longer a file, but an option.

---

I don't know about tags, but if keybindings are in the root folder, you can `initFile = *` and it will use every file in root.
I'm new to nixpkg and I don't have much experience.
I think @rycee will help us